frontend/src/views/PostCarView.js

```diff
@@ -3,21 +3,30 @@ import TextField from "@material-ui/core/TextField";
 import Button from "@material-ui/core/Button";
 import styles from "./PostCarView.css";
 import CarService from "../services/CarService";
+import { ImageUploadView } from "./ImageUploadView";
+import SaveIcon from "@material-ui/icons/Save";
+import DeleteIcon from "@material-ui/icons/Delete";
+import { makeStyles } from "@material-ui/core/styles";
+import PostCarSelectView from "./PostCarSelectView";
```

Comment:
```suggestion
```

Reply:
git status    <-- show files added and not added
git add .    <-- add all files 
git status   <--- check again